src/Page/Index.elm

```diff
@@ -11,11 +12,36 @@ import Shared exposing (Model)
 import Theme.Global exposing (centerContent, mainContainerStyles, pageColumnBlockStyle, pageColumnStyle, topTwoColumnsWrapperStyle)
 
 
+type alias GuideTheme =
+    { name : String
+    , description : String
+    , slug : String
+    }
+
+
+guidesByTheme : List GuideTheme
+guidesByTheme =
+    -- TODO: text changes based on language
+    [ { name = "Alpha", description = "A theme about Alpha things", slug = "alpha" }
+    , { name = "Beta", description = "A theme about Beta things", slug = "beta" }
+    , { name = "Cappa", description = "A theme about Cappa things", slug = "cappa" }
+    ]
+
+
 view : Model -> Html Msg
 view model =
     let
         t =
             translate model.language
+
+        exploreGuideLink : GuideTheme -> Html msg
+        exploreGuideLink guideTheme =
+            a [ href ("/explore/" ++ guideTheme.slug), css [ guideLinkStyle ] ]
+                [ h2 [ css [ guideLinkTitleStyle ] ] [ text guideTheme.name ]
```

Comment:
I think this should be a span not a h2. It's already nested under a h2 and it's just about styling.